docs/imagecustomizer/api/configuration/outputArtifacts.md

```diff
@@ -0,0 +1,45 @@
+---
+parent: Configuration
+---
+
+# outputArtifacts type
+
+Specifies the configuration for the output directory containing the generated
+artifacts, including UKI PE images, shim and systemd-boot.
+
+Example:
+
+```yaml
+artifacts:
```

Comment:
Add the `output` parent object.

Do not give partial examples. Otherwise, it is harder for the user to copy/paste into their config file.